File: python/tamrin2.py
#عددی رو دریافت کرده و مقسوم علیه های آن را چاپ کند
number=input("enter the number:")
number=int(number)
m=1
while m<=number:
    k=number%m
    if k==0:
        print(m)   
    m+=1

File: cpp/if/tamrin2.cpp
#include<iostream>//عددی رو از ورودی دریافت کرده و تشخیص دهد عدد  صفر و یا منفی هست یا مثبت.
using namespace std;
int main()
{
    int num;
    cout<<"enter the number:";
    cin>>num;
    if(num>0)
    cout<<"The number is positive";
    else if(num<0)
    cout<<"The number is negative";
    else
    cout<<"Zero";

    
    return 0;
}

